lib/modules/uscore_v3.1.1/bodyheight_sequence.rb

```diff
@@ -259,6 +265,10 @@ def patient_ids
           reply = perform_search_with_status(reply, search_params) if reply.code == 400
 
           validate_search_reply(versioned_resource_class('Observation'), reply, search_params)
+          resource_returned = fetch_all_bundled_resources(reply, check_for_data_absent_reasons)
+          assert(resource_returned.all? { |resource| ['Observation', 'OperationOutcome'].include? resource.resourceType },
+                 'All resources returned must be of the type Observation or OperationOutcome')
+          resource_returned.reject! { |resource| resource.resourceType == 'OperationOutcome' }
```

Comment:
What does `resource_returned` do?  It doesn't appear to be referenced again, so why clear out OperationOutcomes from it?

Reply:
Ah, I think we have it here because it is needed for MedicationRequest.  But I think we could have the `if` statement in the generator be a little more inclusive so that this line doesn't show up in the generated code for no reason for non-MedicaitonRequests.

Reply:
hmm, in looking at the generator code, I can see why it is a little awkward to have a multi-line if block here.

Reply:
Yeah, I changed it to only add this for medication inclusion. I also just moved the all resources are the right resource check into validate_search_reply